src/OgAccess.php

```diff
@@ -151,14 +153,6 @@ public function userAccess(EntityInterface $group, $operation, AccountInterface
       return $user_access->addCacheableDependency($cacheable_metadata);
     }
 
-    // Update group special permission. At this point, the operation should have
-    // already been handled by Og. If the operation is simply 'edit'
-    // (or 'update' for content entities), it is referring to the current group,
-    // so we have to map it to the special permission.
-    if (in_array($operation, ['update', 'edit'])) {
-      $operation = OgAccess::UPDATE_GROUP_PERMISSION;
-    }
-
```

Comment:
This is contamination that has been added to handle entity operations and translate them to a permission. Since in this PR we will no longer handle operations this can be removed. The test for this is `GroupUpdateTest` which is also being removed in this PR.

This change is also fixing #658